cmake/Modules/Findgrpc.cmake

```diff
@@ -39,12 +34,18 @@ if (NOT grpc_FOUND)
   externalproject_add(grpc_grpc
       GIT_REPOSITORY ${URL}
       GIT_TAG        ${VERSION}
-      CMAKE_ARGS -DgRPC_PROTOBUF_PROVIDER=package -DgRPC_PROTOBUF_PACKAGE_TYPE=CONFIG -DProtobuf_DIR=${EP_PREFIX}/src/google_protobuf-build/lib/cmake/protobuf -DgRPC_ZLIB_PROVIDER=package -DBUILD_SHARED_LIBS=ON
+      CMAKE_ARGS
+        -DgRPC_PROTOBUF_PROVIDER=package
+        -DgRPC_PROTOBUF_PACKAGE_TYPE=CONFIG
+        -DProtobuf_DIR=${EP_PREFIX}/src/google_protobuf-build/lib/cmake/protobuf
+        -DgRPC_ZLIB_PROVIDER=package
+        -DBUILD_SHARED_LIBS=ON
       PATCH_COMMAND ${GIT_EXECUTABLE} apply ${PROJECT_SOURCE_DIR}/patch/fix-protobuf-package-include.patch || true
-      BUILD_BYPRODUCTS ${EP_PREFIX}/src/grpc_grpc-build/grpc_cpp_plugin
-                       ${EP_PREFIX}/src/grpc_grpc-build/${CMAKE_SHARED_LIBRARY_PREFIX}grpc${CMAKE_SHARED_LIBRARY_SUFFIX}
-                       ${EP_PREFIX}/src/grpc_grpc-build/${CMAKE_SHARED_LIBRARY_PREFIX}grpc++${CMAKE_SHARED_LIBRARY_SUFFIX}
-                       ${EP_PREFIX}/src/grpc_grpc-build/${CMAKE_SHARED_LIBRARY_PREFIX}grpc++_reflection${CMAKE_SHARED_LIBRARY_SUFFIX}
+      BUILD_BYPRODUCTS
+        ${EP_PREFIX}/src/grpc_grpc-build/grpc_cpp_plugin
+        ${EP_PREFIX}/src/grpc_grpc-build/${CMAKE_SHARED_LIBRARY_PREFIX}gpr{CMAKE_SHARED_LIBRARY_SUFFIX}
```

Comment:
libgrpc depends on libgpr, so we need to build it.

Reply:
>gpr{CMAKE_SHARED_LIBRARY_SUFFIX}

Changes are looking doubtful. Is `gpr` should be `grpc++` and also what about `$` before brackets.